scripts/render.py

```diff
@@ -0,0 +1,22 @@
+# Usage: python render [image] [json]
+# This file can be used to visualise the tokens in a PDF file.
+
+import cv2
+import json
+import sys
+
+sectiontypes = ["Section","TextBlock","TextRun","Field","Widget","ChoiceGroup","ChoiceField","ChoiceGroupTitle","SectionTitle","Header","HeaderTitle","Image","Footer"]
+sectiontype = "ChoiceGroup"
+img = cv2.imread(sys.argv[1])
+color = (255, 0, 0)
+json_data = json.loads(open(sys.argv[2]).read())
+for j in json_data[sectiontype]:
+    print(j)
```

Comment:
Do we need this print statement?

Reply:
This is the script that Rajiv shared. But good suggestion removed the print statement.